LICENSE

```diff
@@ -0,0 +1,3 @@
+These files are licensed under the terms of the CC0 1.0 Universal license:
```

Comment:
As far as I know, we didn't make a decision on licensing yet, so it's probably to remove it as well.

Reply:
Yeah, sorry, this was picked up from an abandoned project!

Reply:
Er what. I don't think that's great advice @Barthalion. You absolutely cannot fork a project and then just remove the licence. If we don't want something under a particular licence then we need to actually ask the person. In this case @mariospr - it's not terribly hard, he contributes to flatpak and the xdg-portals, he's on the flatpak IRC!

Reply:
Oh duh, good point.

Reply:
How can you even attempt to license a json file, that contains number of URLs to source code tarballs and not even full commands needed to build it all, figured out by tens of distributions so far? How can you distinguish between this file and the one that I could have written from scratch, if it differs only by the lack of the comments and different indentation? So yes, I absolutely think I can remove the license for something that insignificant. There is only so many ways you can write it.

Reply:
@Barthalion That's a loose argument really, as that could be applied even to simple enough software projects and we know Licensing would still apply there.

That said, I'm more than happy for you to drop the current License and use whatever you want instead. That was the whole point I chose "CC0 1.0 Universal License", precisely because I wanted to make sure that anyone could pick up this experiment of mine and do whatever they wanted to do, so please go ahead and change it as you see fit.

Only one thing more, though: as it seems this is now moving from a local "experiment" to being distributed through Flathub, I think it might be worth commenting on https://bugzilla.mozilla.org/show_bug.cgi?id=1290670 about it, so that Mozilla is aware. They might also have some comment on the desired license, too.